check_catkin_lint.sh

```diff
@@ -0,0 +1,21 @@
+# Software License Agreement - BSD License
```

Comment:
If you're going to leave out the bulk of the license at lease be specific which BSD:

```suggestion
# Software License Agreement - BSD 3-Clause License
```

Reply:
I just copied the small disclaimer from the other files. May be we should augment all of them...
Not touching this for now.